### PR TITLE
Editor announcement blog post

### PR DESCRIPTION
**Pull request by lforalaska**

### CODE REVIEW

File: src/posts/editor.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Last week, we launched our [markdown-first](https://contenda.co/blog/markdown) focus blocks. People asked for more frontend focused features, so we're excited to Beta launch our Contenda Editor!
+
+## What does it do?
+
+Glad you asked, let's boogey through our fancy features!
+
+1. Drag 'n' Drop focus blocks
+
+Our content model leans towards giving you *all* the information you need, but we leave it to you to polish. That means some paragraphs might seem similar with slightly different information. Rearrange and delete to your heart's content! If this wasn't convincing enough, here's a ChatGPT poem about it:
```
Comment (jlin99) [inline src/posts/editor.md:21]: isnt this technically a song LOL i dont think poems have a bridge? this is really nitpicky tho lmao

File: src/posts/editor.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+No more limitations, just pure imagination,
+With drag 'n' drop editors, we're building a new foundation.
+
+>Outro:
+So here's to the creators, the dreamers, the innovators,
+With drag 'n' drop editors, we're the orchestrators.
+Editing AI-generated content, a magical combination,
+A new era of design, let's start a celebration.
+
+![Demo gif of drag 'n' drop](/blogimages/DragAndDrop.gif)
```
Comment (jlin99) [inline src/posts/editor.md:59]: i dont know if i _love_ that the chatGPT song kind of interrupts the explanation + the gif but its kinda cute and fun. can we try putting the gifs in between the numbered point and the paragraph? e.g.
```
2. Images, pictures, screenshots

![Demo gif of images](/blogimages/images.gif)

Let's put all the pretty pictures...
```

Comment (jlin99) [reply src/posts/editor.md:59]: this way the chatGPT song won't be as distracting

**Review by jlin99 — APPROVED** [review]
lgtm otherwise

File: src/posts/editor.md (hunk 2 above)
Comment (GradySalzman) [reply src/posts/editor.md:59]: I like this idea as well!

**Pull request merged**